### PR TITLE
Add option for fixed size of the calendar

### PR DESCRIPTION
**Pull request by alextkachuk**
Lets to set fixed size of the calendar (6 rows of weeks). In addition, lets to prevent issue with top position of the calendar.

Without fixed size:
react-datepicker-before

Fixed size:
react-datepicker-fixed

### CODE REVIEW

**Comment by rafeememon** [comment]
Can we call this flag `fixedHeight` instead?

File: src/datepicker.jsx [hunk 1]
```diff
@@ -149,7 +150,8 @@ var DatePicker = React.createClass({
         includeDates={this.props.includeDates}
         showYearDropdown={this.props.showYearDropdown}
         todayButton={this.props.todayButton}
-        outsideClickIgnoreClass={outsideClickIgnoreClass} />
+        outsideClickIgnoreClass={outsideClickIgnoreClass}
+        isFixed={this.props.isCalendarFixed} />
```
Comment (rafeememon) [inline src/datepicker.jsx]: I think we should probably call the prop the same thing through all of the components, `fixedHeight`

**Comment by alextkachuk** [comment]
@rafeememon Yes, sure.

File: src/datepicker.jsx (hunk 1 above)
Comment (alextkachuk) [inline src/datepicker.jsx]: @rafeememon - ok

**Comment by alextkachuk** [comment]
@rafeememon Done

File: src/month.jsx [hunk 2]
```diff
@@ -33,7 +34,7 @@ var Month = React.createClass({
     const startOfMonth = this.props.day.clone().startOf('month').startOf('week')
     return [0, 1, 2, 3, 4, 5]
       .map(offset => startOfMonth.clone().add(offset, 'weeks'))
-      .filter(startOfWeek => this.isWeekInMonth(startOfWeek))
+      .filter(startOfWeek => this.props.fixedHeight ? true : this.isWeekInMonth(startOfWeek))
```
Comment (rafeememon) [inline src/month.jsx]: This can be simplified to `this.props.fixedHeight || this.isWeekInMonth(startOfWeek)`

Comment (alextkachuk) [inline src/month.jsx]: @rafeememon I'm agree, done.

**Comment by omgaz** [comment]
LGTM 👍

**Comment by rafeememon** [comment]
@alextkachuk thanks for the PR!

Re #361, #510

**Comment by flyalex** [comment]
thanks for this fix!

**Pull request merged**